desktop/apps/artwork/components/auction/view.coffee

```diff
@@ -7,7 +7,7 @@ openBuyersPremiumModal = require './components/buyers_premium/index.coffee'
 AuthModalView = require '../../../../components/auth_modal/view.coffee'
 inquire = require '../../lib/inquire.coffee'
 acquire = require '../../lib/acquire.coffee'
-helpers = require './helpers.coffee'
+ViewHelpers = require './helpers.coffee'
```

Comment:
Sounds good 👍 Just note that as a rule of thumb, keeping the import var and the filename 1:1 (if not destructuring) is a best practice that will reduce complexity in the long run.